deploy.sh

```diff
@@ -1,16 +1,17 @@
 #!/usr/bin/env nix-shell
 #!nix-shell -i bash ./shell.nix
 
-set -eux
+set -euxo pipefail
 
 PROJECT_NAME=bonk-api
 
 nix build .#packages.x86_64-linux.dockerImage
+# note: will write auth token to XDG_RUNTIME_DIR
+flyctl auth token | skopeo login -u x --password-stdin registry.fly.io
 skopeo \
```

Comment:
It's... kinda rude that we can't `--password-stdin` with `skopeo copy` :/ Then we could prevent the token from hitting the disk at all.

Reply:
Yeah. I didn't want to deal with suggesting, implementing and waiting for a merge on the feature upstream though :sweat_smile: